catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/transformer/TransformerDescriptorApplication.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.transformer;
+
+import static org.codice.ddf.catalog.ui.transformer.TransformerDescriptors.getTransformerDescriptor;
+import static spark.Spark.get;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.Map;
+import spark.servlet.SparkApplication;
+
+public class TransformerDescriptorApplication implements SparkApplication {
+
+  private TransformerDescriptors descriptors;
+
+  private static final Gson GSON =
+      new GsonBuilder().disableHtmlEscaping().serializeNulls().create();
+
+  private static final Map<String, String> TRANSFORMER_NOT_FOUND_RESPONSE =
+      new ImmutableMap.Builder<String, String>().put("message", "Transformer not found").build();
+
+  private static final Map<String, String> TRANSFORMER_TYPE_NOT_FOUND_RESPONSE =
+      new ImmutableMap.Builder<String, String>()
+          .put("message", "Transformer type not found")
+          .build();
+
+  private static final String QUERY_RESPONSE_TRANSFORMER = "query";
+
+  private static final String METACARD_TRANSFORMER = "metacard";
+
+  @Override
+  public void init() {
+    get("/transformers/metacard", (req, res) -> GSON.toJson(descriptors.getMetacardTransformers()));
```

Comment:
✏️ Consider using the three-arg version to separate the endpoint logic from the response format: `get("/transformers/metacard", (req, res) -> descriptors.getMetacardTransformers(), GSON::toJson)`.